packages/pitayasmoothie/README.md

```diff
@@ -35,6 +35,15 @@ Highlight color: ![#321f7ad9](https://via.placeholder.com/20/321f7ad9/321f7ad9.p
 | ![#f45c7b](https://via.placeholder.com/20/f45c7b/f45c7b.png) | `#f45c7b` | 5.7 : 1  | AA          | AAA        |
 | ![#ffe46b](https://via.placeholder.com/20/ffe46b/ffe46b.png) | `#ffe46b` | 14.2 : 1 | AAA         | AAA        |
 
+## Font
```

Comment:
After a further inspection this should be in the main README, chances are users will not dig and read all the other nested READMEs and this will come as a surprise when changing any of the themes